alembic/versions/0809bdb23579_ading_user_id.py

```diff
@@ -0,0 +1,32 @@
+"""ading user_id.
```

Comment:
It will be good to add a proper comment when creating this file as it is useful.